src/pages/registrationPage/registrationPage.tsx

```diff
@@ -0,0 +1,150 @@
+      toast.success('Account has been created');
+      navigate('/');
+    } catch (err: any) {
+      const error = err.response?.data.message;
+      toast.error(error.toString());
+    }
+  };
+
+  //Handle for change state of visible password
+  const handleCheckboxChange = () => {
+    setShowPassword(!showPassword);
+  };
+  const handleCheckboxAdminChange = () => {
+    setIsAdmin(!isAdmin);
+  }
+
+  return (
+    <section className="container-registration">
+      <div className="reg-title-container">
+        <Title headerText="Create your account" />
+      </div>
+      <form onSubmit={handleSubmit(registrationHandler)}>
+        <Input
+          type="text"
+          placeholder="username"
+          aboveInputText="Username"
+          name="username"
+          register={register}
+          errors={errors}
+          validation={{
+            required: 'Username is required',
+            minLength: {
+              value: 5,
+              message: 'Username must be at least 5 characters',
+            },
+            maxLength: {
+              value: 30,
+              message: 'Username cannot exceed 30 characters',
+            },
+          }}
+        />
+        <Input
+          type="email"
+          placeholder="user@gmail.com"
+          aboveInputText="Email"
+          name="email"
+          register={register}
+          errors={errors}
+          validation={{
+            required: 'Email is required',
+            minLength: {
+              value: 5,
+              message: 'Email must be at least 5 characters',
+            },
+            maxLength: {
+              value: 30,
+              message: 'Email cannot exceed 30 characters',
+            },
+            pattern: {
+              value: /@/,
+              message: "Email must contain '@'",
+            },
+          }}
+        />
+        <Input
+          type={showPassword ? 'text' : 'password'}
+          placeholder="password"
+          aboveInputText="Password"
+          name="password"
+          register={register}
+          errors={errors}
+          validation={{
+            required: 'Password is required',
+            minLength: {
+              value: 5,
+              message: 'Password must be at least 5 characters',
+            },
+            maxLength: {
+              value: 30,
+              message: 'Password cannot exceed 30 characters',
+            },
+          }}
+        />
+        <Input
+          type={showPassword ? 'text' : 'password'}
+          placeholder="confirm password"
+          aboveInputText="Confirm Password"
+          name="confirmPassword"
+          register={register}
+          errors={errors}
+          validation={{
+            required: 'Confirm Password is required',
+            validate: (value: string) =>
+              value === getValues('password') || 'Passwords do not match',
+          }}
+        />
+        <Checkbox
+          className="checkbox-field"
```

Comment:
nstead of hardcoding the class names for the checkboxes, you might consider dynamically generating the class names based on the checkbox text or another identifier. This can make your component more flexible and easier to maintain.

Reply:
you did not resolve that

Reply:
What does it mean? Use some library for that or what?